### PR TITLE
master-lps-104193

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-108100
https://issues.liferay.com/browse/LPS-108171
https://issues.liferay.com/browse/LPS-108169
https://issues.liferay.com/browse/LPS-108091
https://issues.liferay.com/browse/LPS-108097

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro [hunk 1]
```diff
@@ -71,6 +71,25 @@ definition {
...
+	macro deleteContentViaBulkEditing {
+		if (isSet(recycleBinDisabled)) {
+			ClickNoError(locator1 = "Icon#DELETE");
+
+			AssertConfirm(value1 = "Are you sure you want to delete the selected entries? They will be deleted immediately.");
+
+			Alert.viewSuccessMessage();
+		}
+		else {
+			ClickNoError(locator1 = "Icon#DELETE");
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro]: @Jeremy-Ch  We should keep the trash and delete icons separate. Please create an Icon#TRASH locator and apply that to locators which have a trash icon in a separate LRQA ticket.

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/brianchandotcom/liferay-portal/pull/84488<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/5220e82486ad~3...5220e82486ad)

**Pull request closed without merging**